src/test/java/org/opensearch/jobscheduler/utils/JobDetailsServiceIT.java

```diff
@@ -16,102 +18,115 @@
 import org.mockito.Mockito;
 import org.opensearch.action.ActionListener;
 import org.opensearch.cluster.service.ClusterService;
+import org.opensearch.jobscheduler.model.JobDetails;
 import org.opensearch.test.OpenSearchIntegTestCase;
 
 public class JobDetailsServiceIT extends OpenSearchIntegTestCase {
 
     private ClusterService clusterService;
+    private Set<String> indicesToListen;
+
+    private String expectedJobIndex;
+    private String expectedJobType;
+    private String expectedJobParamAction;
+    private String expectedJobRunnerAction;
+    private String expectedExtensionUniqueId;
+
+    private String expectedDocumentId;
+    private String updatedJobIndex;
 
     @Before
     public void setup() {
         this.clusterService = Mockito.mock(ClusterService.class, Mockito.RETURNS_DEEP_STUBS);
-        Mockito.when(this.clusterService.state().routingTable().hasIndex("opensearch-plugins-job-details"))
+        Mockito.when(this.clusterService.state().routingTable().hasIndex(JobDetailsService.JOB_DETAILS_INDEX_NAME))
             .thenReturn(false)
             .thenReturn(true);
```

Comment:
I'm just now coming up to speed on Mockito, but I'm confused about these consecutive `thenReturn()` statements.  This doesn't look right to me but it might be. Is the second one kind of like "else"?

Reply:
Each `thenReturn` corresponds to a single invocation of `clusterService.state().routingTable().hasIndex()`. The first invocation of this method will return `false`, and the second invocation will return `true`